apache/maven.md

```diff
@@ -1,3 +1,63 @@
 [Voltar](intro.md)
 
-# Apache Maven
\ No newline at end of file
+# Apache Maven
+
+## O que é o Maven
```

Comment:
Como quase todo o texto é baseado na referência [1], sugiro que lá no início vocês digam que esse texto foi escrito com base em [1], assim vocês não precisam ficar repetindo a referência ao longo do texto. E onde for diferente, por exemplo [2], você coloca a referência.

Reply:
Feito.